app/assets/stylesheets/credits.scss

```diff
@@ -274,6 +283,10 @@
       font-weight: bold;
     }
 
+    @media screen and ( min-width:380px ) {
+      padding: 10px 2existing-credits0px;
```

Comment:
This looks lit might be a typo. Do you mind fixing this before I take a closer look?

Reply:
Wow, yes, I fixed